docs/sphinx/features.rst

```diff
@@ -4,4 +4,6 @@
 Features and Examples
 =====================
 
+test
```

Comment:
It would be better to try and keep commits like this out of the branch so have a cleaner history when we merge 😄